src/encoded/static/components/datacolors.js

```diff
@@ -1,3 +1,6 @@
+import _ from 'underscore';
+
```

Comment:
All of these changes are directly from Encode: I copied the file from Encode over to Regulome.